src/components/legal-notice/Sidebar.js

```diff
@@ -1,54 +1,51 @@
 import React from 'react';
-import {
-  Sidebar,
-  SidebarSection,
-  SidebarSectionList,
-  SidebarItem,
-} from 'components/Sidebar';
+import { Sidebar, SidebarSection, SidebarItem } from 'components/Sidebar';
 
 const LegalNoticeSidebar = () => (
   <Sidebar sticky={true} side="left">
     <SidebarSection>
-      <strong>Legal Notices</strong>
+      <strong className="pl-3">Legal Notices</strong>
     </SidebarSection>
 
-    <SidebarSectionList>
-      <SidebarItem
-        to="/legal-notices/terms-of-service/"
-        text="Terms of Service & DPA"
-        partiallyActive={true}
-      />
-
-      <SidebarItem
-        to="/legal-notices/evaluation-licence/"
-        text="Evaluation Licence"
-        partiallyActive={true}
-      />
-
-      <SidebarItem
-        to="/legal-notices/sub-processors/"
-        text="List of Sub-Processors"
-        partiallyActive={true}
-      />
-
-      <SidebarItem
-        to="/legal-notices/privacy-notice/"
-        text="Privacy Notice"
-        partiallyActive={true}
-      />
-
-      <SidebarItem
-        to="/legal-notices/cookies-policy/"
-        text="Cookies Policy"
-        partiallyActive={true}
-      />
-
-      <SidebarItem
-        to="/legal-notices/acceptable-use-policy/"
-        text="Acceptable Use Policy"
-        partiallyActive={true}
-      />
-    </SidebarSectionList>
+    <SidebarSection>
+      <ul className="p-0 list-none">
```

Comment:
The `<SidebarSectionList />` no longer takes `children` so we have to use a plain `<SidebarSection />`.